packages/design-foundation/package.json

```diff
@@ -76,18 +76,19 @@
     "url": "https://github.com/gohypergiant/standard-toolkit"
   },
   "scripts": {
-    "build": "pnpm generate && pnpm license && pnpm format && tsdown",
```

Comment:
i remember why we were formatting here. it's because we wanted to format the generated code.

i added `pnpm run format` to the `generate` script. alternatively, we could add explicit biome/prettier format calls inside the generate scripts themselves.

we probably want this generation to be pre-build though, right?